portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,22 +37,16 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private int numComments = 10;
-  private int page = 0;
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
       Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
       PreparedQuery results = datastore.prepare(query);
-      int offset = page * numComments;
-      FetchOptions options = FetchOptions.Builder.withLimit(numComments).offset(offset);
```

Comment:
I think you should still have this,  but numComments and offset should be retrieved from the request object. I think it's good to include the total number of comments in the response, but the response should only include the comments we care about seeing, not all the comments.